app/components/base/__tests__/HeroImage.test.tsx

```diff
@@ -0,0 +1,13 @@
+import renderer from "react-test-renderer";
+import HeroImage, { StyledHero } from "../HeroImage";
+
+test("New Hero Image", () => {
+  const tree = renderer.create(<HeroImage />).toJSON();
+
+  expect(tree).toMatchSnapshot();
+});
+
+test("Style", () => {
+  const tree = renderer.create(<StyledHero />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
To test attributes + styles on styled component, we'll have to use jest-styled-components, or a different testing library. What do you think?  toHaveStyle() won't work here.

Reply:
Ah, actually the snapshot testing that we're doing with these components accounts for testing the styles as well! [Here's ](https://medium.com/styled-components/effective-testing-for-styled-components-67982eb7d42b) a good article I found that explains snapshotting!